content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: Graduation of Device Manager to GA!'
+date: 2021-11-21
+slug: graduation-of-devicemanager-to-GA
+---
+
+**Author:** Swati Sehgal (Red Hat)
+
+##  Quick Intro to the Device Plugin framework
+
+### Introduction
```

Comment:
```suggestion
```

We avoid starting articles with a heading. It's OK to jump straight in to introductory text.

Please adjust other headings to match.